### DISCUSSION TITLE
Define return value

### DISCUSSION THREAD

**Discussion by zhu-maofeng** [WalletConnect/walletconnect-monorepo#3639] [Developer Support] [answered]
I am using web3modal. May I ask how to edit the confirmation chain prompt after the user approves it,Like the image below, there will be a chain prompt after the same link
![1f5cbbd4f5984aeb57380a7ee84b080](https://github.com/WalletConnect/walletconnect-monorepo/assets/121092649/6cf0d211-ea33-42ca-9c29-bee2186181b2)

**Comment by glitch-txs** [comment, marked as answer]
I think that's the description Metatag of your website

**Reply by zhu-maofeng** [reply]
thinks
